app/javascript/store/index.js

```diff
@@ -14,5 +15,6 @@ export default new Vuex.Store({
     scores,
     statistics,
     admin,
+    froggo_team,
```

Comment:
Identifier 'froggo_team' is not in camel case    camelcase

Reply:
Acá tampoco se si tiene sentido cambiarle el nombre a la carpeta solo para que no tire este error, qué hago ?